js/script.js

```diff
@@ -139,16 +139,18 @@
 			});
 		}
 
-		$cpuFooterInfo.text(t('serverinfo', 'Load average') + ": " + cpu1 + " (" + t('serverinfo', 'Last minute') + ")");
+		$cpuFooterInfo.text(t('serverinfo', 'Load average') + ": " + cpu1 + " (" + t('serverinfo', 'last minute') + ")");
```

Comment:
This should be transformed into
```suggestion
		$cpuFooterInfo.text(t('serverinfo', 'Load average {cpu} (last minute)', { cpu: cpu1 }));
```